src/v2/api/controllers/AuthController.ts

```diff
@@ -46,24 +46,37 @@ export class AuthController {
     private userService: UserService,
   ) {}
 
-  @ApiBasicAuth()
+  @ApiBearerAuth()
   @ApiOkResponse({
     type: AuthLoginResponse,
   })
   @ApiUnauthorizedResponse({
     description: `\n
+    UnauthorizedException:
+      Unauthorized
     UnauthorizedException:
       The email hasn't verified yet`,
   })
-  @UseGuards(LocalAuthGuard)
+  @ApiEndpoint({
+    summary: 'Login to the user account',
+    guards: LocalAuthGuard,
+  })
   @Post('/login')
   async login (@Request() req) {
     return this.authService.login(req.user);
   }
 
   @ApiBearerAuth()
   @ApiOkResponse()
-  @UseGuards(TelegramGuard)
+  @ApiUnauthorizedResponse({
```

Comment:
add ApiBadRequestResponse with these validation messages
`Token cannot be empty`
`Telegram id cannot be empty`